docs/build-integration-nft-data.md

```diff
@@ -29,13 +29,18 @@ The NFT trading platform serves the storage needs of NFT files by integrating Cr
 
 ### 1. Import NFT files into IPFS
 
-With the NFT platform running an IPFS node first, the platform can import NFT files into IPFS when NFT is generated.
+Import NFT files into IPFS by Crust IPFS Gateway when NFT is generated
+
+- Select an optimal [Crust IPFS Gateway Endpoint](https://github.com/crustio/crust-apps/blob/master/packages/apps-config/src/ipfs-gateway-endpoints/index.ts)
+
+- Use the following default W3Auth Header or Get a new IPFS W3Auth Header by [buildIPFSWeb3AuthGW](https://wiki.crust.network/docs/en/buildIPFSWeb3AuthGW#usage)
```

Comment:
- Use the following default Web3 Auth Header or create your own Web3 Auth Header by refering this doc: